lib/dom-patcher.js

```diff
@@ -41,6 +45,9 @@ export class DOMPatcher {
     this.el = document.createElement(tagName);
     if (classMatches) {
       this.el.className = classMatches.map((c) => c.slice(1)).join(` `);
+      // this attribute setting ensures that svg elements behave as expected and will ensure
+      // compatibility with different snabbdom versions
+      this.el.setAttribute(`class`, this.el.classList);
```

Comment:
this was prompted by `emptyNodeAt` changes.

Original: https://github.com/snabbdom/snabbdom/blob/v0.7.4/src/snabbdom.ts#L63
New: https://github.com/snabbdom/snabbdom/blob/v3.0.3/src/init.ts#L87